src/main/java/com/google/api/codegen/viewmodel/ServiceDocView.java

```diff
@@ -40,11 +41,17 @@ public static Builder newBuilder() {
   }
 
   public String firstLine() {
-    return lines().get(0);
+    if (lines().size() > 0) {
+      return lines().get(0);
+    }
+    return "";
   }
 
   public List<String> remainingLines() {
-    return lines().subList(1, lines().size());
+    if (lines().size() > 0) {
+      return lines().subList(1, lines().size());
+    }
+    return Collections.<String>emptyList();
```

Comment:
Putting more logic like this in the view model will make it more difficult to just store view models as data later (if we decide to do so). Could we move the firstLine/remainingLine logic somewhere else? There are two other view models that have this pattern too (`MapParamDocView` and `SimpleParamDocView`).